defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+gocd_common_user: go
+gocd_common_userid: 5000
```

Comment:
I don't see this and `gocd_common_groupid` being used. Can we remove this? Also please go through all the existing variables and remove the one's we don't use anywhere within the role.